tests/test_output_measurements.py

```diff
@@ -23,15 +22,14 @@ def setUp(self):
         metric2 = Metric("{}.thing2".format(package), 'thing2', u.mag)
         self.metric_set = MetricSet("testing", [metric1, metric2])
 
-    # def tearDown(self):
-    #     if os.path.exists(self.outfile):
-    #         os.remove(self.outfile)
-
+    @unittest.skip('FIXME DM-8477 not implemented yet')
```

Comment:
That's fair, though it's probably more 'FIXME DM-8477 going to probably disappear...'